README.md

```diff
@@ -61,18 +61,27 @@ Go to http://localhost:8000/admin and create a new user.
 
 ## Access API
 
-### Examples for students  
+### Auth
+
+Go to http://localhost:8000/api/users/register 
+
+As you can see below, you can fill the body form-data with info:   
+
+![](https://github.com/zetonteam/zeton_django/blob/auth/images/postman_register_01.png?raw=true)  
+
+After that you can hit http://localhost:8000/api/users/students/ with GET method and put into Headers key and value.
+
+Then you have access to private endpoint.   
+
+![](https://github.com/zetonteam/zeton_django/blob/auth/images/postman_register_02.png?raw=true)  
+
```

Comment:
Could you explain why we must paste prefix `JWT` in token When we send a post request?

Reply:
This is similar to how you put a token in Postman using the Authorization -> and selecting Type -> here we have to select 'JWT', but I don't see 'JWT' on the list, so I passed it to the Headers.

Bearer Token doesn't work here, I don't know why, if I find a moment I'll read about it.